doc/website/blog/2022-12-20-wilco-atlas-quest.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Learning to solve migration issues in a realistic simulation game"
+authors: rotemtam
+tags: [simulation, learning, wilco]
+image: https://atlasgo.io/uploads/images/explore_twitter_card.png
+---
+
+### Learning new things 
+
+There are many ways to learn about a new topic. You can read books, watch videos, or even take a course. But one of the
+most effective ways to learn is by doing. This is especially true when it comes to learning about a new tool or
+technology. In order to truly understand how a tool works, you need to use it and see how it behaves in different
+situations.
+
+This is why I'm excited to announce that we've partnered with [Wilco](https://trywilco.com/) to create a new
+kind of learning experience for Atlas users. Wilco develops a platform, built to emulate the conditions at a tech
+startup, sends users on "quests" that cover everyday engineering tasks - from deploying an app to finding the root cause
+of a production issue - utilizing real-life tech stacks.
+
+When I first heard about Wilco, I was immediately intrigued by the idea of using a simulation game to
+teach people about database migrations and especially about solving all kinds of issues that many engineers
```

Comment:
```suggestion
teach people about database migrations. This platforms allows engineers to learn how to solve all kinds of issues that many 
```

Reply:
I will split into two but the stress here isn't on the platform but on the opportunity to practice "disastrous scenarios"